extensions/ssh-manager/src/create.tsx

```diff
@@ -0,0 +1,37 @@
+import { ActionPanel, Form, showHUD, SubmitFormAction, useNavigation } from "@raycast/api";
+import { ISSHConnection } from "./types";
+import { getConnections, saveConnections } from "./storage.api";
+import { createId } from "./utils";
```

Comment:
```suggestion
import { createId } from "./utils";
```
Could be unused if you change